Libraries/Lists/VirtualizedList.js

```diff
@@ -921,19 +912,14 @@ class VirtualizedList extends React.PureComponent<Props, State> {
       onScrollBeginDrag: this._onScrollBeginDrag,
       onScrollEndDrag: this._onScrollEndDrag,
       onMomentumScrollEnd: this._onMomentumScrollEnd,
+      contentContainerStyle: inversionStyle,
```

Comment:
`this.props.contentContainerStyle` is lost, please use array of styles or maybe `StyleSheet.compose` (I am not sure what is the difference).